plugins/tile/tile-ipc.hpp

```diff
@@ -215,6 +215,38 @@ inline std::unique_ptr<tile::tree_node_t> build_tree_from_json(const wf::json_t&
     return root;
 }
 
+inline wf::json_t handle_ipc_set_show_maximized(const wf::json_t& params)
+{
+    if (!params.has_member("view-id"))
```

Comment:
No need for these checks. `json_get_uint64` will throw an exception if view-id is not found and it will be handled in the ipc method repository. Same for the other `.has_member('show-maximized')` check below.